torch/optim/adam.py

```diff
@@ -105,6 +105,8 @@ class Adam(Optimizer):
         capturable (bool, optional): whether this instance is safe to capture in a CUDA graph.
             Passing True can impair ungraphed performance, so if you don't intend to
             graph capture this instance, leave it False (default: False)
+        differentiable (bool,optional): use adaptative calculation of the derivative of 
+            learning rate, setting to true can affect performance (default: False) 
```

Comment:
Thanks for adding the slight note--let's be more descriptive in that it would impair (negatively affect) performance.
```suggestion
            learning rate. Setting to True can impair performance, so leave it False if you don't intend to run autograd through this instance. (default: False) 
```